tools/aquainfra_importer/aquainfra_importer.xml

```diff
@@ -0,0 +1,13 @@
+<tool id="aquainfra_importer" name="AquaINFRA Importer" tool_type="data_source" version="1.0" profile="22.05">
```

Comment:
can you split this please into several PRs? Or is this just a bad branch with things that don't belong together?

Reply:
oh, that is a mistake